lib/eu/hydrologis/smash/mainview.dart

```diff
@@ -100,7 +102,8 @@ class MainViewWidgetState extends State<MainViewWidget> with WidgetsBindingObser
     _mapController = MapController();
     mapState.mapController = _mapController;
 
-    int noteInGpsMode = GpPreferences().getIntSync(KEY_DO_NOTE_IN_GPS, POINT_INSERTION_MODE_GPS);
+    int noteInGpsMode = GpPreferences()
```

Comment:
Again, for some reasons a line break got into here. There are more down the line.